wallet/src/components/Modals/Assets/ReceiveModal/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './styles.scss';
+import PropTypes from 'prop-types';
+
+const ReceiveModal = ({ handleClose, show }) => {
+  return (
+    show && (
+      <div id="my-modal" className="modal">
+        <div className="modal-content">
+          <div className="modal-header">
+            <span className="close">&times;</span>
+            <h2>Modal Header</h2>
+            <button onClick={() => handleClose(false)}>exit</button>
+          </div>
+          <div className="modal-body">
+            <p>This is my modal</p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla repellendus nisi, sunt
```

Comment:
What is this file?

Reply:
I was testing to create a modal from scratch. Will remove it.